pkg/codegen/nodejs/gen_test.go

```diff
@@ -59,6 +60,51 @@ func TestGeneratePackage(t *testing.T) {
 				}
 			},
 		},
+		{
+			"Simple schema with enum types",
+			filepath.Join("schema", "simple-enum-schema.json"),
+			false,
+			func(files map[string][]byte) {
+				assert.Contains(t, files, "rubberTree.ts")
+				assert.Contains(t, files, "types/input.ts")
+				assert.Contains(t, files, "types/enums.ts")
+				assert.Contains(t, files, "types/index.ts")
+
+				for fileName, file := range files {
+					if fileName == "rubberTree.ts" {
```

Comment:
Can we instead keep the expected contents around as either files or literal strings?